### DISCUSSION TITLE
Longer Scoreboard

### DISCUSSION THREAD

**Discussion by chrisoflaherty** [riffnshred/nhl-led-scoreboard#364] [General]
Is it possible to make the scoreboard longer with the raspberry pi or is it not powerful enough.

**Comment by riffnshred** [comment]
Yes. You can chain the display and then in your command, instead of using col=64, you do col=128. You can even do bigger and all if you want but I dont give support yet for different size other then 64x32. I believe if you do 128x32, it works as good, but on lets say 128x64, the layout on some boards does not work. It is in the todo list to officially support those,  but Im working on other things these days.
